x-pack/test/api_integration/apis/features/features/features.ts

```diff
@@ -135,6 +135,60 @@ export default function ({ getService }: FtrProviderContext) {
           ].sort()
         );
       });
+
+      it('should return a full feature set with correct scope', async () => {
+        const { body } = await supertest.get('/api/features').set('kbn-xsrf', 'xxx').expect(200);
```

Comment:
__nit:__ `get` is a "safe" method, so, technically it shouldn't require `kbn-xsrf`
```suggestion
        const { body } = await supertest.get('/api/features').expect(200);
```